src/haddock/libs/libpdb.py

```diff
@@ -252,8 +249,7 @@ def get_pdb_file_suffix_variations(file_name, path=None, sep="_"):
         If no files are found return an empty list.
     """
     basename = Path(file_name)
-    path = path or Path.cwd()
-    return list(path.glob(f"{basename.stem}{sep}*{basename.suffix}"))
+    return list(Path(".").glob(f"{basename.stem}{sep}*{basename.suffix}"))
```

Comment:
Sorry, I am struggling to get free time these weeks. Some comments follow:

1. If we need to change this function specifically for `topoaa`, could it be safer to make a new function only for `topoaa`? In order to have one that returns absolute paths and another that returns relative paths'. I remember editing this function some time ago to get some functionalities; that `path=None` is not innocent. I believe.
2. functions like `get_result_or_same_in_list` and `transform_to_list` are likely impossible to remove. Because, in many cases, we can have arguments that are strings or lists. For example, `molecules=path` or molecules=[path1, path2]`; yet the functionality needs to be maintained. Yes, we can use the unpacking in python functions (`*`), but it is troublesome to handle that all around the code. I can say with almost 100% sure that if the `get_result_or_same_in_list` is because there are some situations where it was needed.

Reply:
Yes, I see. Perfect! I believe that part of the code can be improved and cleaned. Especially after `pdb-tools` lib-like functionalities were developed afterwards. Go ahead with this change. I see there's confidence; I just wanted to call attention when changing from relative to absolute paths. Because HADDOCK3 is meant to run locally and on grid controlling paths becomes tricky also because of the need to be moving the CWD across steps as they run.

:+1: merge! :wink: